test/integration/backward_compatible/HazelcastClientTest.js

```diff
@@ -80,22 +80,16 @@ configParams.forEach((cfg) => {
         let client;
         let managed;
 
-        before(async function () {
```

Comment:
Why do we need to make it `before/afterEach`?

Reply:
Actually I don't know how this test was passing but, first running `getDistributedObjects returns all dist objects` and then `getDistributedObjects does not return removed object` causes names in the second test to be like this: 
```
Array(4) [map,
set,
map2,
map3]
```

So they affect each other. Plus, our new test shuts down the client so we need to create another one for other tests